hdate/htables.py

```diff
@@ -25,7 +25,7 @@
         0, 0, 0, 0, 11, 0, 0, 0, 0, 0,
         0, 0, 0, 0, 0, 0, 0, 0, 0, 33],
     [  # Adar
-        0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0, 0, 0, 0, 0, 34, 0, 0, 0,
```

Comment:
It is already in the table: 
יום הזיכרון לחללי מערכות ישראל שמקום קבורתם לא נודע